contributing/docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:18.04
```

Comment:
Would using CentOS be a better match here, since we are using Red Hat Enterprise Linux (RHEL) for BFD servers?